### PR TITLE
[deployments-k8s#2992] Switch scripts

### PR DESCRIPTION
**Pull request by Bolodya1997**
## Description
Adds `to-local.sh` and `to-ref.sh` switching between `local` and `ref` modes.

**Update:** adds `to-export.sh` for sharing local branch with someone else.
**Update:** adds `refcheck` CI check.

## Issue
https://github.com/networkservicemesh/deployments-k8s/issues/2992

## How Has This Been Tested?

- [ ] Added unit testing to cover
- [x] Tested manually
- [ ] Tested by integration testing
- [ ] Have not tested

## Types of changes

- [ ] Bug fix
- [x] New functionality
- [ ] Documentation
- [ ] Refactoring
- [ ] CI

### CODE REVIEW

**Comment by edwarnicke** [comment]
Please add shellcheck to the ci.yaml if we are including shell scripts.

File: examples/heal/remote-nsmgr-death/README.md [hunk 1]
```diff
@@ -239,10 +239,10 @@ kubectl exec ${NEW_NSE} -n ${NAMESPACE} -- ping -c 4 172.16.1.103
 
 Restore NSMgr setup:
 ```bash
-kubectl apply -k github.com/networkservicemesh/deployments-k8s/apps/nsmgr?ref=050a5817cb8104b907e2c241a7af38c08d3102d1 -n nsm-system
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/apps/nsmgr -n nsm-system?ref=050a5817cb8104b907e2c241a7af38c08d3102d1
```
Comment (glazychev-art) [inline examples/heal/remote-nsmgr-death/README.md:242]: Please, check the script.
"ref=" is not at the end of the line here

Comment (Bolodya1997) [reply examples/heal/remote-nsmgr-death/README.md]: Thank you, fixed.

**Review by denis-tingaikin — CHANGES_REQUESTED** [review]
1. Could you respond on https://github.com/networkservicemesh/deployments-k8s/issues/2992#issuecomment-930267417?
2. Could you create an example of how to use it in .md format for external developers?

**Comment by Bolodya1997** [comment]
> 2. Could you create an example of how to use it in .md format for external developers?

Added a new paragraph in `README.md`.

**Comment by edwarnicke** [comment]
@Bolodya1997 LGTM... just needs a rebase.

**Review by denis-tingaikin — APPROVED** [review]

**Review by denis-tingaikin — APPROVED** [review]

**Pull request merged**